compact/tof_endcap.xml

```diff
@@ -7,13 +7,13 @@
       --------------------------
     </comment>
     <comment> Forward/backward TOF position </comment>
-    <constant name="ForwardTOF_length"           value="min(ForwardTOFRegion_length, 2.*cm)"/>
-    <constant name="ForwardTOF_zmin"             value="ForwardTOFRegion_zmin + ForwardTOFRegion_length - ForwardTOF_length - 1*cm"/>
-    <constant name="ForwardTOF_nlayers"          value="2"/>
+    <constant name="ForwardTOF_length"           value="15*cm"/>
+    <constant name="ForwardTOF_zmin"             value="163.0*cm"/>
+    <constant name="ForwardTOF_nlayers"          value="1"/>
     <constant name="ForwardTOFMod_offset"        value="1.2*cm"/>
-    <constant name="BackwardTOF_length"          value="min(BackwardTOFRegion_length, 2.*cm)"/>
-    <constant name="BackwardTOF_zmin"            value="BackwardTOFRegion_zmin + 0.5*cm"/>
-    <constant name="BackwardTOF_nlayers"         value="2"/>
+    <constant name="BackwardTOF_length"          value="10*cm"/>
+    <constant name="BackwardTOF_zmin"            value="166.0*cm"/>
+    <constant name="BackwardTOF_nlayers"         value="1"/>
```

Comment:
Please do not change parametrization with hard-coded values.

Reply:
Done